docs/components/Link/Link.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from "@storybook/addon-docs";
+
+# Link
+
+<Meta title="Components/Text and Typography/Link/Docs" />
+
+The Link will render an `<a>` tag with the appropriate `href` and `target`
+attributes, when/as applicable.
+
+## Usage guidelines
+
+A Link marked external will always open in a new tab. Set this when linking away
+from Jobber or to other applications or sites across Jobber where this behaviour
+is appropriate (e.g. linking from Jobber Online to the Jobber Help Center).
+
+Link should not be used to submit form data, trigger actions, or alter the state
+of the page. Use a [Button](../?path=/docs/components-actions-button-docs--page)
+for these purposes.
+
+## Content guidelines
+
+Contents of the link should be textual, and may include some formatting features
+(e.g. italic text for emphasis may be appropriate, changing the inherited
+underline probably isn't). The text can be plain, from a `Typography` component,
```

Comment:
I think we should make this reference to the `typography component a link to that section of the documentation to follow the above pattern like we did with button.

Reply:
I agree, good idea!